packages/patrol/example/android/app/build.gradle

```diff
@@ -55,8 +55,15 @@ android {
     }
 
     testOptions {
-        reportDir "$rootDir/test-reports"
-        resultsDir = "$rootDir/test-results"
+        def targetPath = findProperty("target")
+        if (targetPath != null) {
```

Comment:
What about the `else`? Is the default sufficient?

Reply:
We're okay with `target` being `null` - in such case, the default (`lib/main.dart`) is used.